src/components/OcBreadcrumb.vue

```diff
@@ -13,7 +13,7 @@
         >
           {{ item.text }}
         </oc-button>
-        <span v-else :aria-current="getAriaCurrent(index)" v-text="item.text" />
+        <span v-else :aria-current="getAriaCurrent(index)" tabindex="-1" v-text="item.text" />
```

Comment:
A negative value doesn't allow sequential keyboard navigation to the el. Is that what we want? If yes, I find the changelog a bit confusing as it says "make it focusable for keyboard users"

Reply:
You're totally right it's a contradiction and the changelog item does not get the point across well. Fixing it, thanks for the sharp eye 🕵🏽